physionet-django/user/forms.py

```diff
@@ -535,6 +534,8 @@ def clean_reference_email(self):
             if reference_email in self.user.get_emails():
                 raise forms.ValidationError("""You can not put yourself
                     as a reference.""")
+            if reference_email.split('@')[-1] in ["yahoo.com", "163.com", "126.com", "outlook.com", "gmail.com", "qq.com", "foxmail.com"]:
```

Comment:
We should move this validation rule to one of the validation modules (`validators.py`). Ideally we would have something like an `is_institutional_email()` validator that you would call here.